src/io/iohk/scalanet/test/peergroup/InetPeerGroupUtilsSpec.scala

```diff
@@ -0,0 +1,42 @@
+package io.iohk.scalanet.peergroup
+
+import java.net.{Inet4Address, Inet6Address, InetAddress, InetSocketAddress}
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.concurrent.ScalaFutures._
+import org.scalatest.{BeforeAndAfterAll, FlatSpec}
+import org.scalatest.Matchers._
+
+import scala.concurrent.duration._
+class InetPeerGroupUtilsSpec extends FlatSpec with BeforeAndAfterAll {
+
+  implicit val patienceConfig: ScalaFutures.PatienceConfig = PatienceConfig(5 seconds)
+
+  behavior of "getChannelId"
+
+  it should "create a channelId for ip6address" in {
```

Comment:
Please use a table and remove copy/paste: http://www.scalatest.org/user_guide/table_driven_property_checks

Reply:
Please review again resolved changed to Table driven